### PR TITLE
fix: add matthewraymond

### PR DESCRIPTION
**Pull request by matthewraymondh**
## Changes proposed

I am contributing to linkfree by adding my profile in it!

[x] - Correct; marked as done
[X] - Correct; marked as done

[ ] - Not correct; marked as **not** done
-->

## Check List (Check all the applicable boxes) 

- [X] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [X] I have updated the documentation accordingly.
- [X] All new and existing tests passed.
- [X] This PR does not contain plagiarized content.
- [X] The title of my pull request is a short description of the requested changes.

## Screenshots

image

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/839"></a>

### CODE REVIEW

File: public/data/matthewraymond.json [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    "milestones": [
+      {
+        "title": "Started College",
+        "date": "August 2021",
+        "icon": "book",
+        "color": "grey",
+        "description": "Started college"
+      }
+    ]
+}
```
Comment (Panquesito7) [inline public/data/matthewraymond.json:42]: Missing an endline.
```suggestion
}

```

File: public/data/matthewraymond.json [hunk 2]
```diff
@@ -0,0 +1,42 @@
+{
+    "name": "Matthew Raymond",
+    "type": "personal",
+    "bio": "I am a web and software developer, designer, open source contributor, and a student.",
+    "avatar": "https://github.com/YOUR_GITHUB_USERNAME.png",
```
Comment (Panquesito7) [inline public/data/matthewraymond.json:5]:
```suggestion
    "avatar": "https://github.com/matthewraymondh.png",
```

File: public/data/matthewraymond.json [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+        "url": "https://twitter.com/MatthewRay03",
+        "icon": "twitter"
+      },
+      {
+        "name": "Follow me on Instagram",
+        "url": "https://www.instagram.com/_matthewraymond_",
+        "icon": "instagram"
+      },
+      {
+        "name": "My LinkedIn profile",
```
Comment (Panquesito7) [inline public/data/matthewraymond.json:23]:
```suggestion
        "name": "Connect with me on LinkedIn",
```

File: public/data/matthewraymond.json [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+        "url": "https://www.linkedin.com/in/matthew-raymond-915310176/",
+        "icon": "linkedin"
+      },
+      {
+        "name": "My portfolio",
```
Comment (Panquesito7) [inline public/data/matthewraymond.json:28]:
```suggestion
        "name": "Check out my portfolio",
```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

**Comment by matthewraymondh** [comment]
Thanks for the suggestion, updated my code as your suggestion!

File: public/data/matthewraymond.json [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+    "milestones": [
+      {
+        "title": "Started College",
+        "date": "August 2021",
+        "icon": "book",
+        "color": "grey",
+        "description": "Started college"
+      }
+    ]
+}
```
Comment (krishguptadev) [inline public/data/matthewraymond.json:42]:
```suggestion
{
  "name": "Matthew Raymond",
  "type": "personal",
  "bio": "I am a web and software developer, designer, open source contributor, and a student.",
  "avatar": "https://github.com/matthewraymondh.png",
  "links": [
    {
      "name": "Follow me on GitHub",
      "url": "https://github.com/matthewraymondh",
      "icon": "github"
    },
    {
      "name": "Follow me on Twitter",
      "url": "https://twitter.com/MatthewRay03",
      "icon": "twitter"
    },
    {
      "name": "Follow me on Instagram",
      "url": "https://www.instagram.com/_matthewraymond_",
      "icon": "instagram"
    },
    {
      "name": "Connect with me on LinkedIn",
      "url": "https://www.linkedin.com/in/matthew-raymond-915310176/",
      "icon": "linkedin"
    },
    {
      "name": "Check out my portfolio",
      "url": "https://www.matthewraymond.me/",
      "icon": "globe"
    }
  ],
  "milestones": [
    {
      "title": "Started College",
      "date": "August 2021",
      "icon": "book",
      "color": "grey",
      "description": "Started college"
    }
  ]
}

```

**Review by krishguptadev — APPROVED** [review]

**Comment by matthewraymondh** [comment]
Thanks for the suggestion, updated my code as your suggestion!

File: public/data/matthewraymond.json [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+    {
+      "title": "Started College",
+      "date": "August 2021",
+      "icon": "book",
+      "color": "grey",
+      "description": "Started college"
+    }
+  ]
+}
+
```
Comment (krishguptadev) [inline public/data/matthewraymond.json:43]:
```suggestion
{
  "name": "Matthew Raymond",
  "type": "personal",
  "bio": "I am a web and software developer, designer, open source contributor, and a student.",
  "avatar": "https://github.com/matthewraymondh.png",
  "links": [
    {
      "name": "Follow me on GitHub",
      "url": "https://github.com/matthewraymondh",
      "icon": "github"
    },
    {
      "name": "Follow me on Twitter",
      "url": "https://twitter.com/MatthewRay03",
      "icon": "twitter"
    },
    {
      "name": "Follow me on Instagram",
      "url": "https://www.instagram.com/_matthewraymond_",
      "icon": "instagram"
    },
    {
      "name": "Connect with me on LinkedIn",
      "url": "https://www.linkedin.com/in/matthew-raymond-915310176/",
      "icon": "linkedin"
    },
    {
      "name": "Check out my portfolio",
      "url": "https://www.matthewraymond.me/",
      "icon": "globe"
    }
  ],
  "milestones": [
    {
      "title": "Started College",
      "date": "August 2021",
      "icon": "book",
      "color": "grey",
      "description": "Started college"
    }
  ]
}

```

**Review by matthewraymondh — COMMENTED** [review]
I changed my code as suggested

**Comment by krishguptadev** [comment]
> I changed my code as suggested

Hmm it isn't updated plus the Prettier CI is failing

**Comment by matthewraymondh** [comment]
can you suggest me how to fix it? btw i allready commited ur suggestion.

File: public/data/matthewraymond.json [hunk 7]
```diff
@@ -0,0 +1,42 @@
...
+  "milestones": [
+    {
+      "title": "Started College",
+      "date": "August 2021",
+      "icon": "book",
+      "color": "grey",
+      "description": "Started college"
+    }
+  ]
+}
```
Comment (Pradumnasaraf) [inline public/data/matthewraymond.json:42]:
```suggestion
}

```

**Review by Pradumnasaraf — CHANGES_REQUESTED** [review]

**Comment by matthewraymondh** [comment]
Fixed the issue!

**Review by Pradumnasaraf — APPROVED** [review]

**Review by matthewraymondh — COMMENTED** [review]
did all these aswell!

**Review by krishguptadev — APPROVED** [review]

**Review by Panquesito7 — APPROVED** [review]
LGTM 🚀 Thank you for your contribution! 😄👍

**Pull request merged**